File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PlantBatchesCreatePlantBatchWasteRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PlantBatchesCreatePlantBatchWasteRequestItem(
    @JsonProperty("mixedMaterial")
    val mixedMaterial: String? = null,
    @JsonProperty("note")
    val note: String? = null,
    @JsonProperty("plantBatchName")
    val plantBatchName: String? = null,
    @JsonProperty("reasonName")
    val reasonName: String? = null,
    @JsonProperty("unitOfMeasureName")
    val unitOfMeasureName: String? = null,
    @JsonProperty("wasteDate")
    val wasteDate: String? = null,
    @JsonProperty("wasteMethodName")
    val wasteMethodName: String? = null,
    @JsonProperty("wasteWeight")
    val wasteWeight: Double? = null
)

File: sdks/thunkmetrc-go/wrapper/models/plant_batches_create_plant_batch_waste_request_item.go
package models

type PlantBatchesCreatePlantBatchWasteRequestItem struct {
    MixedMaterial string `json:"MixedMaterial,omitempty"`
    Note string `json:"Note,omitempty"`
    PlantBatchName string `json:"PlantBatchName,omitempty"`
    ReasonName string `json:"ReasonName,omitempty"`
    UnitOfMeasureName string `json:"UnitOfMeasureName,omitempty"`
    WasteDate string `json:"WasteDate,omitempty"`
    WasteMethodName string `json:"WasteMethodName,omitempty"`
    WasteWeight float64 `json:"WasteWeight,omitempty"`
}
